packages/util-package-finder/lib/index.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
+
+const _ = require('lodash/fp');
+const orderBy = require('lodash/orderBy');
+const fetch = require('node-fetch');
+const semver = require('semver');
+
+/**
+ * API endpoint to NPM registry
+ * @type {String}
+ */
+const npmRegistry = 'https://registry.npmjs.org';
```

Comment:
Do we want to support passing in the registry so that this can be used on private repos? But default it to `npmjs.org`

Reply:
yeah, we could do, nice idea